assets/airports/loww.json

```diff
@@ -386,448 +362,339 @@
+              "MASUR": ["_MASUR272009", ["MASUR", "A170-|S330"], ["TEMTA", "A60-|S250"], ["NERDU", "A70+|S250-"], ["WW987", "A70+|S250-"], ["WW985", "A70|S250-"], ["WW983", "A70|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "LANUX": ["_LANUX331017", ["LANUX", "A150-|S330"], ["NERDU", "A70+|S250-"], ["WW987", "A70+|S250-"], ["WW985", "A70|S250-"], ["WW983", "A70|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "GAMLI": ["LIMRA", ["GAMLI", "A60+|S330+"], ["BARUG", "A170-|S330"], ["WW450", "A90-|S250-"], ["BALAD", "A100|S250-"], ["WW988", "A80|S250-"], ["WW982", "A50-|S250-"], ["WW980", "A60-|S250-"], ["WW978", "A50-|S250-"], ["WW976", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "XANUT": ["_NIGSI202010", ["NIGSI", "A180-|S330"], ["BALAD", "A100|S250-"], ["WW988", "A80|S250-"], ["WW982", "A50-|S250-"], ["WW980", "A60-|S250-"], ["WW978", "A50-|S250-"], ["WW976", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "NATEX": ["_NATEX080019", ["NATEX", "A140-|S330-"], ["PESAT", "A60+|S250-"], ["WW991", "A70|S250-"], ["WW989", "A60+|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "MIKOV": ["_MIKOV345020", ["MIKOV", "A170-|S330"], ["MABOD", "A60+|S250-"], ["WW987", "A100|S250-"], ["WW985", "A80|S250-"], ["WW983", "A80|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "REKLU": ["_REKLU082028", ["REKLU", "A180|S250-"], ["MABOD", "A60+|S250-"], ["WW987", "A100|S250-"], ["WW985", "A80|S250-"], ["WW983", "A80|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]]
+            },
+          "body": []
         }
     },
     "spawnPatterns": [
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Vienna_International_Airport), the annual traffic count is 226,811, which would equate to an overall average of 25.89 movements per hour. Thus, the traffic in the sim should fall between 25.8acph and 77.7acph.

Currently, there are 78acph. Please reduce to something in the realm of 45acph to 55acph, unless there is evidence to support that it experiences long periods of light traffic (and therefore markedly higher volumes during the heavy times).